dlib/dnn/layers.h

```diff
@@ -147,14 +147,16 @@ namespace dlib
         template <typename SUBNET>
         void forward(const SUBNET& sub, resizable_tensor& output)
         {
+            conv.setup(sub.get_output(),
+                       filters(params,0),
+                       _stride_y,
+                       _stride_x,
+                       padding_y_,
+                       padding_x_);
+            
             conv(output,
                 sub.get_output(),
-                filters(params,0),
-                _stride_y,
-                _stride_x,
-                padding_y_,
-                padding_x_
-                );
+                filters(params,0));
```

Comment:
Oops, never mind.  Funny formatting in the diff I'm looking at.  Obviously this line needs to be here :)